### DISCUSSION TITLE
Just just want to find out why I didn't get any ZK tokens.

### DISCUSSION THREAD

**Discussion by SnapcatSol** [zkSync-Community-Hub/zksync-developers#597] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

If you could check my wallet i did contribute for a long time.

https://nftcopilot.com/zksync-rank-check?address=0xaa5e283e81be5802d29a1ff410044500da19caa6

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

As a note, this space if for developer questions, please join the Discord for further support: https://join.zksync.dev/
